### PR TITLE
Add raise exception on serializer validation

### PR DESCRIPTION
**Pull request by no-dap**
re-created PR
Refs : [#797](https://github.com/graphql-python/graphene-django/pull/797)

### CODE REVIEW

**Comment by no-dap** [comment]
## Overview
Screen Shot 2019-10-19 at 1 35 50 PM
  
Errors didn't appended to ExecutionResult's errors if we don't raise errors,  
they just exists inside data.  

Screen Shot 2019-10-19 at 1 00 25 PM

We can request errors as a result query, but this looks super weird.  
  
## Another issue
DjangoFormMutation, SerializerMutation with invalid inputs return status code 200.  
I opened another issue [#804](https://github.com/graphql-python/graphene-django/issues/804)

**Comment by no-dap** [comment]
hello there?

File: graphene_django/rest_framework/tests/test_mutation.py [hunk 1]
```diff
@@ -204,20 +205,23 @@ class Meta:
             serializer_class = MySerializer
 
     # missing required fields
-    result = MyMutation.mutate_and_get_payload(None, mock_info(), **{})
-    assert len(result.errors) > 0
+    with raises(ValidationError):
+        result = MyMutation.mutate_and_get_payload(None, mock_info(), **{})
+        assert len(result.errors) > 0
```
Comment (zbyte64) [inline graphene_django/rest_framework/tests/test_mutation.py]: Does this assert ever get called? If I understand correctly line 209 should raise a `ValidationError` thus line 210 never gets called.

Comment (no-dap) [reply graphene_django/rest_framework/tests/test_mutation.py]: No it doesn't. I think line 210 is unnecessary indeed.

**Pull request closed without merging**